src/core/etl/src/Flow/ETL/Transformer/StyleConverter/RenameStrategy.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Transformer\StyleConverter;
+
+enum RenameStrategy : string
```

Comment:
that's not exactly what I had in mind, since it does not allow for any paramters and transliterate would need one. 
Also it moves problem just to another place, instead of having methods now we have enum cases. 

I was thinking about making it more sophisticated where `RenameStrategy` is an interface and each of the below can be it's own implementation. 

This way it can be used like this: 

```
df()->renameEach(rename_transliterate("..."))
df()->renameEach(rename_style(Style::LOWER_CASE))
df()->renameEach(rename_style(Style::UCFIRST))
```